pathod/language/base.py

```diff
@@ -105,7 +109,7 @@ def __repr__(self):
 class _TokValueLiteral(Token):
 
     def __init__(self, val):
-        self.val = val.decode("string_escape")
+        self.val = escaped_str_to_bytes(str(val))
```

Comment:
These sort of errors come up: https://travis-ci.org/mitmproxy/mitmproxy/jobs/132823626#L460-L516

The function receives `unicode` values as input, but I can't figure out the source.